diff --git a/extractor_unix.go b/extractor_unix.go
index f4f2746..8992579 100644
--- a/extractor_unix.go
+++ b/extractor_unix.go
@@ -12,7 +12,7 @@ import (

 func lchmod(name string, mode os.FileMode) error {
        var flags int
-       if runtime.GOOS == "linux" {
+       if runtime.GOOS == "linux" || runtime.GOOS == "zos" {
                if mode&os.ModeSymlink != 0 {
                        return nil
                }
@@ -29,6 +29,12 @@ func lchmod(name string, mode os.FileMode) error {
 }

 func lchtimes(name string, mode os.FileMode, atime, mtime time.Time) error {
+    if runtime.GOOS == "zos" {
+        if err := lchmod(name, mode); err != nil {
+            return err
+        }
+    }
+
        at := unix.NsecToTimeval(atime.UnixNano())
        mt := unix.NsecToTimeval(mtime.UnixNano())
        tv := [2]unix.Timeval{at, mt}
